webui/react/src/components/MultiSortMenu.tsx

```diff
@@ -51,6 +51,10 @@ export const optionsByColumnType = {
     { label: 'Newest → Oldest', value: 'desc' },
     { label: 'Oldest → Newest', value: 'asc' },
   ],
+  [V1ColumnType.ARRAY]: [
```

Comment:
we wanna exclude array type in filtering and sorting. do we still need it?

Reply:
I needed to add the array element to avoid react build errors. Updated so the list key is empty